.github/ISSUE_TEMPLATE/developer-site-bug-report.yml

```diff
@@ -1,4 +1,4 @@
-name: API Bug Report
+name: Developer Site Bug Report
 description: A bug has been found in Discord's API, Gateway, OAuth2, SDK, etc.
```

Comment:
Shouldn't the description needs to be updated too?

Reply:
💀 I will edit it

Reply:
🤦🏻 I promise I know what I'm doing 🤣

Reply:
I don't 😁